hrm-domain/lambdas/export-json-writer/src/index.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import format from 'date-fns/format';
+import type { SQSBatchResponse, SQSEvent, SQSRecord } from 'aws-lambda';
+import { putS3Object } from '@tech-matters/s3-client';
+import {
+  EntityNotification,
+  getNormalisedNotificationPayload,
+  isCaseNotification,
+} from './entityNotification';
+import { getSsmParameter } from '@tech-matters/ssm-cache';
+import { getTwilioAccountSidFromHrmAccountId } from '@tech-matters/types/dist/HrmAccountId';
```

Comment:
This import will cause issues I think (`@tech-matters/types/dist/HrmAccountId`).